README.md

```diff
@@ -26,8 +26,9 @@ will generate a JavaScript wrapper class file roughly like this:
 
 ```js
 const conversions = require("webidl-conversions");
-const impl = require("./utils.js").implSymbol;
-const ctorRegistry = require("./utils.js").ctorRegistrySymbol;
+const utils = require("./utils.js");
+
+const { implSymbol, ctorRegistrySymbol } = utils;
```

Comment:
`utils` doesn't seem to be used anywhere. (This is an example so we don't have to do the exact same as the output.)

```suggestion
const { implSymbol, ctorRegistrySymbol } = require("./utils.js");
```